CMakeLists.txt

```diff
@@ -27,50 +31,61 @@ if (CCACHE_FOUND)
 	set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
 endif()
 
-find_package(ZLIB QUIET)
-if (NOT ZLIB_FOUND)
-    add_subdirectory(3rdparty/zlib EXCLUDE_FROM_ALL)
-    set(ZLIB_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/zlib" "${CMAKE_CURRENT_BINARY_DIR}/3rdparty/zlib" CACHE INTERNAL "")
-    set(ZLIB_LIBRARY zlibstatic)
+if(WIN32)
+	add_definitions(-DUNICODE)
+	add_definitions(-D_WIN32_WINNT=0x0602)
```

Comment:
This is windows 8. Can we make it 10?

Reply:
Should actually be Windows 7 for compatibility?

Reply:
It was originally 0x0601 (Win7) but I had trouble getting it to build because some xinput structs weren't defined. What value visual studio project file uses?

Reply:
Hmm it's targeting 8.1 so should be safe